src/test/java/com/wikia/webdriver/common/remote/context/RemoteContext.java

```diff
@@ -0,0 +1,12 @@
+package com.wikia.webdriver.common.remote.context;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.openqa.selenium.WebDriver;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.WebDriver'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)